packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -370,7 +370,7 @@ const TESTS = new Map([
     ])],
     ['html', [
       { test: 'include', expected: configHref('subscriptionPrivacyUrl', 'subscription-payment-expired', 'subscription-privacy') },
-      { test: 'include', expected: configHref('subscriptionSettingsUrl', 'subscription-payment-expired', 'cancel-subscription', 'plan_id', 'product_id', 'uid', 'email') },
+      { test: 'include', expected: configHref('subscriptionSettingsUrl', 'subscription-payment-expired', 'update-billing', 'plan_id', 'product_id', 'uid', 'email') },
```

Comment:
This change and the two below are related to the Map to array change affecting existing tests.